File: WPServer/WPProjectServer/WPProjectServer/DatabaseObjects/CopyStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WPProjectServer.DatabaseObjects
{
    public class CopyStatus : DatabaseObject
    {
        public Copy Copy { get; set; }

        public User RentBy { get; set; }

        public DateTime RentTime { get; set; }

        public DateTime ReturnTime { get; set; }

        public double DaysDelay { get; set; }
    }
}

File: WPWeb/Library/src/main/java/models/CopyStatus.java
package src.main.java.models;

import java.util.Date;

/**
 * Created by Szymon on 2015-10-13.
 */
public class CopyStatus extends DatabaseObject
{
        public Copy copy;

        public User rentBy;

        public Date rentTime;

        public Date returnTime;

        public double daysDelay;
}